include/mbedtls/config.h

```diff
@@ -1453,7 +1453,7 @@
  *
  * TODO: Document
  */
-#define MBEDTLS_SSL_USE_MPS
+//#define MBEDTLS_SSL_USE_MPS
```

Comment:
Could you keep MPS enabled by default?

Reply:
The tests work fine if I enable MPS but compilation with Visual Studio fails. There is some include file that is missing.

Reply:
After I included the sources from library/mps I ran into compilation errors with the 

> __attribute__((unused))

statement since it is not supported by Visual Studio. Maybe we can replace this with a macro that uses this statement for GCC but not for Visual Studio?

Reply:
Currently, the only compile-time issues that exists with Visual Studio is in line 655 of mps.c indicating "warning C4003: not enough arguments for function-like macro invocation 'MBEDTLS_MPS_ERROR_INFO'" with this line of code:

> /* Replicates code from above for each MPS error - see error.h. */
>     MBEDTLS_ERR_MPS_ERROR_LIST;

A few syntax errors will then follow...

Reply:
I suspect this is because MSVC isn't standard compliant w.r.t. macro evaluation, but haven't had the time yet to investigate further.

Reply:
It is fine to postpone this issue. The code compiles nicely using non-Visual Studio environments and the tests pass as well.

Reply:
And: this issue is unrelated to the ticket handling as well